parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -62,11 +62,11 @@ ERR_AZURE_STACK_GET_NETWORK_CONFIGURATION=121 {{/* Error fetching the network co
 ERR_AZURE_STACK_GET_SUBNET_PREFIX=122 {{/* Error fetching the subnet address prefix for a subnet ID */}}
 
 {{/* BCC/BPF-related error codes */}}
-ERR_IOVISOR_KEY_DOWNLOAD_TIMEOUT=126 {{/* Timeout waiting to download IOVisor repo key */}}
-ERR_IOVISOR_APT_KEY_TIMEOUT=127 {{/* Timeout waiting for IOVisor apt-key */}}
-ERR_BCC_INSTALL_TIMEOUT=128 {{/* Timeout waiting for bcc install */}}
-ERR_BPFTRACE_BIN_DOWNLOAD_FAIL=129 {{/* Failed to download bpftrace binary */}}
-ERR_BPFTRACE_TOOLS_DOWNLOAD_FAIL=130 {{/* Failed to download bpftrace default programs */}}
+ERR_IOVISOR_KEY_DOWNLOAD_TIMEOUT=132 {{/* Timeout waiting to download IOVisor repo key */}}
```

Comment:
technically 1 - 2, 126 - 165, and 255 are all reserved so we should avoid that whole range

Reply:
missed that all the way up to 165 was reserved for signals. I didn't realize there were so many! fixed, thanks.